logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/SelfServerConfigUseCase.kt

```diff
@@ -19,31 +19,37 @@
 package com.wire.kalium.logic.feature.user
 
 import com.wire.kalium.common.error.CoreFailure
+import com.wire.kalium.common.functional.fold
 import com.wire.kalium.logic.configuration.server.ServerConfig
 import com.wire.kalium.logic.configuration.server.ServerConfigRepository
 import com.wire.kalium.logic.data.user.UserId
-import com.wire.kalium.common.functional.fold
 
 /**
  * This use case is responsible for retrieving the current user's server configuration.
  */
-class SelfServerConfigUseCase internal constructor(
-    private val selfUserId: UserId,
-    private val serverConfigRepository: ServerConfigRepository
-) {
+interface SelfServerConfigUseCase {
```

Comment:
Unrelated/related, making it an interface, so it can be tested/mocked from users of this class